Makefile

```diff
@@ -43,6 +43,24 @@ test:
 test-integration: 
 	./scripts/run-integration-tests.sh 2>&1 | tee test-integration.log
 
+# Instead of building from the local checked-out source, this will install
+# the most recent commit with a release tag. Use the most recent tagged release
+# for production swaps.
+.PHONY: build-release
+build-release:
+	go install -tags=prod github.com/athanorlabs/atomic-swap/cmd/...@latest
+	mkdir -p bin
+	mv $(GOPATH)/bin/swapd $(GOPATH)/bin/swapcli $(GOPATH)/bin/bootnode bin/
```

Comment:
why not just build directly into bin/ instead of moving?

Reply:
Brain flop on my part. Fixed. I had worked on that multi-platform script recently and you can't set GOBIN when doing a cross-platform build, but since this is a native build, that restriction does not apply.